sympy/physics/continuum_mechanics/cable.py

```diff
@@ -0,0 +1,334 @@
+
+    @property
+    def right_support(self):
+        """
+        Returns the position of the right support.
+        """
+        return self._right_support
+
+    @property
+    def loads(self):
+        """
+        Returns the magnitude and direction of the loads
+        acting on the cable.
+        """
+        return self._loads
+
+    @property
+    def loads_position(self):
+        """
+        Returns the position of the loads acting on the
+        cable.
+        """
+        return self._loads_position
+
+    @property
+    def length(self):
+        """
+        Returns the length of the cable.
+        """
+        return self._length
+
+    @property
+    def reaction_loads(self):
+        """
+        Returns the reaction forces at the supports.
+        """
+        return self._reaction_loads
+
+    def add_supports(self, *args):
+        """
+        This method adds the right and left supports to the Cable object.
+
+        Parameters
+        ==========
+        Pass two tuples of the form (label, x, y) as inputs, where the first
+        tuple represents the left support, while the later represents the
+        right support. Each element in these tuples are:
+
+        label: String or a symbol
+            The label for the support.
+
+        x: Sympifyable
+            The x-coordinate of the position of the support.
+
+        y: Sympifyable
+            The y-coordinate of the position of the support.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.cable import Cable
+        >>> c = Cable()
+        >>> c.add_supports(('A', 3, 4), ('B', 4, 5))
+        >>> c.supports
+        {'A': [3, 4], 'B': [4, 5]}
+        >>> c.left_support
+        [3, 4]
+        >>> c.right_support
+        [4, 5]
+        """
+        if len(self._supports) != 0:
+            raise ValueError("Supports already exist")
+
+        elif len(args) != 2:
+            raise ValueError("Pass only two arguments: left support and right support")
+
+        elif args[0][0] == args[1][0]:
+            raise ValueError("Supports can not have the same label")
+
+        elif args[0][1] == args[1][1] and args[0][2] == args[1][2]:
+            raise ValueError("Supports can not be at the same position")
+
+        elif args[0][1] > args[1][1]:
+            raise ValueError("The x coordinate of left support should be less than its right support counterpart")
+
+        else:
+            for i in args:
+                label = i[0]
+                x = sympify(i[1])
+                y = sympify(i[2])
+
+                self._supports[label] = [x, y]
+
+                if len(self._left_support) == 0:
+                    self._left_support.append(x)
+                    self._left_support.append(y)
+
+                else:
+                    self._right_support.append(x)
+                    self._right_support.append(y)
+
+                self._support_labels.append(label)
```

Comment:
While applying a support you should also apply variable reaction forces in X and Y direction. You can use the `apply_load` method for this.  These variables will be calculated later by the solve method. See how beam and truss implement this:

https://github.com/sympy/sympy/blob/9d4517c6b1840aa2c2dfd1617aea32ccedc7041d/sympy/physics/continuum_mechanics/truss.py#L576

https://github.com/sympy/sympy/blob/9d4517c6b1840aa2c2dfd1617aea32ccedc7041d/sympy/physics/continuum_mechanics/beam.py#L455